packages/react-native-reanimated/RELEASE.md

```diff
@@ -0,0 +1,138 @@
+    - Select your branch as the destination for the workflow:
+      ![image](https://hackmd.io/_uploads/SJMK9HFTye.png 'title')
+
+13. Wait for the workflow to finish - it should take about 2 minutes.
+14. Select your workflow from the list and download artifacts.
+    ![image](https://hackmd.io/_uploads/B1MZoSYayl.png)
+
+15. Unzip your artifacts:
+
+    - `unzip react-native-reanimated-x.y.z.tgz.zip`
+    - **Do not** double-click the file to extract it as it would extract it from both `zip` and `tgz` levels.
+
+16. Test the package in a clean React Native app:
+
+    - `npx @react-native-community/cli init MyApp --skip-install`
+    - `cd MyApp`
+    - `yarn add ~/Downloads/react-native-reanimated-x.y.z.tgz`
+    - Add `plugins: ['react-native-reanimated/plugin']` in `babel.config.js` - as in our [Getting Started](https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/getting-started/) guide,
+    - Copy [Bokeh Example](https://github.com/software-mansion/react-native-reanimated/blob/main/apps/common-app/src/apps/reanimated/examples/BokehExample.tsx) to `App.tsx`,
+    - Open and build `ios/MyApp.xcworkspace` on iOS,
+    - Run `open -a 'Android Studio'` and build on Android,
+    - Start Metro using `yarn start --reset-cache`.
+
+17. Test the package in a clean React Native Web app:
+
+    - `npx create-expo-app my-app`
+    - Set up using **Getting Started** tutorial from [Expo Docs](https://docs.expo.dev/workflow/web/),
+    - `yarn add ~/Downloads/react-native-reanimated-x.y.z.tgz`
+    - If you don't have `babel.config.js`, create it using [this instruction](https://docs.expo.dev/versions/latest/config/babel/),
+    - Add `plugins: ['react-native-reanimated/plugin']` in `babel.config.js` - as in our [Getting started](https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/getting-started/) guide,
+    - Copy [Bokeh Example](https://github.com/software-mansion/react-native-reanimated/blob/main/apps/common-app/src/apps/reanimated/examples/BokehExample.tsx) to `App.tsx` - when using **expo-router** find and copy to `index.tsx`,
+    - Run dev: `npx expo start --web`,
+    - Run release: `npx expo export -p web && npx serve dist --single`.
+
+18. If something doesn't work you can fix it in this or in a separate PR (depending on the case), and repeat the steps.
+19. If everything works fine you can merge the PR ✅.
+20. Upload the package to NPM:
+
+    - Check package size and file list with `npm publish --dry-run react-native-reanimated-x.y.z.tgz` - see if there is anything suspicious,
+    - Publish package with `npm publish react-native-reanimated-x.y.z.tgz`,
+    - If you don't have permission to `react-native-reanimated` on NPM ask **Krzysztof Magiera** (or one of the RNOS members with permission, currently: Krzysztof Piaskowy, Tomasz Zawadzki, Bartłomiej Błoniarz, Tomasz Żelawski, Mateusz Łopaciński, Patrycja Kalińska).
```

Comment:
Let's just remove this bullet point as currently we don't plan to grant npm access to everyone who asks.
```suggestion
```